share/spack/setup-env.sh

```diff
@@ -245,8 +245,8 @@ if [ "${need_module}" = "yes" ]; then
     if [ "${_sp_module_prefix}" != "not_installed" ]; then
         #activate it!
         export MODULE_PREFIX=${_sp_module_prefix}
-        _spack_pathadd PATH "${MODULE_PREFIX}/Modules/bin"
-        module() { eval `${MODULE_PREFIX}/Modules/bin/modulecmd ${SPACK_SHELL} $*`; }
+        _spack_pathadd PATH "${MODULE_PREFIX}/bin"
+        module() { eval `${MODULE_PREFIX}/bin/modulecmd ${SPACK_SHELL} $*`; }
     fi;
```

Comment:
It might make sense to keep this backwards-compatible, that is, test whether `Modules/bin` exists and use it if it does.

Reply:
`Modules/bin` no longer exists with modules version 4. A solution would be to create a symlink.

Reply:
It should also be possible to first check whether bin exists and only if it does not, fall back to Modules/bin, right?